lua/telescope-code-actions/utils/init.lua

```diff
@@ -42,13 +42,17 @@ M.get_code_actions = function()
 		local result = response.result
 		local error = response.err
 
+		if error ~= nil and error["code"] == -32601 then
```

Comment:
I think this nil check is redundant
also replace `error["code"]` with `error.code`

Reply:
The nil check is not redundant, as it would throw an index error when error is nil
Fixed, thanks.